routers/web/repo/render.go

```diff
@@ -5,21 +5,16 @@
 package repo
 
 import (
-	"bytes"
-	"fmt"
-	"io"
 	"net/http"
 	"path"
 
-	"code.gitea.io/gitea/modules/charset"
 	"code.gitea.io/gitea/modules/context"
 	"code.gitea.io/gitea/modules/git"
 	"code.gitea.io/gitea/modules/markup"
-	"code.gitea.io/gitea/modules/typesniffer"
 	"code.gitea.io/gitea/modules/util"
 )
 
-// RenderFile renders a file by repos path
+// RenderFile uses an external render to render a file by repos path
```

Comment:
This is not right. It's not always an external render

Reply:
I do not think the old code is right.

That are many reasons that the old code is not right, but the most important one is:

Does the old code work as expected? You rendered some contents by internal render, but where are the styles?